### PR TITLE
Update discussion-group.md

### PR DESCRIPTION
**Pull request by musingvirtual**
Fixing time zone chaos and updating the dates for future ones.

**Preview**: https://deploy-preview-3175--opentelemetry.netlify.app/community/end-user/discussion-group/

### CODE REVIEW

**Review by mhausenblas — APPROVED** [review]
LGTM

File: content/en/community/end-user/discussion-group.md [hunk 1]
```diff
@@ -32,22 +32,22 @@ Here are upcoming sessions, or you can view them on the
 [OpenTelemetry calendar](https://github.com/open-telemetry/community#calendar):
 
 - **EMEA (Europe, Middle East, and Africa)**: every third Tuesday of the month
-  at 11AM CET (GMT +1), join
+  at 11AM GMT, join
   [here](https://us06web.zoom.us/j/85691064809?pwd=c0VCejh)
```
Comment (chalin) [inline content/en/community/end-user/discussion-group.md:36]:
```suggestion
  at 11AM GMT ([Zoom link](https://us06web.zoom.us/j/85691064809?pwd=c0VCejh))
```

File: content/en/community/end-user/discussion-group.md [hunk 2]
```diff
@@ -32,22 +32,22 @@ Here are upcoming sessions, or you can view them on the
...
-  - June 20
-  - July 18
   - August 15
+  - September 19
+  - October 17
 - **APAC (Asia Pacific)**: every third Wednesday of the month at 11AM India ST
   (GMT +5.5), register [here](https://lu.ma/1w129wgu) to get the Zoom link
```
Comment (chalin) [inline content/en/community/end-user/discussion-group.md:41]:
```suggestion
- **APAC (Asia Pacific)**: every third Wednesday of the month at 11AM India ST
  (GMT +5.5), [register](https://lu.ma/1w129wgu) to get the Zoom link
```

File: content/en/community/end-user/discussion-group.md [hunk 3]
```diff
@@ -32,22 +32,22 @@ Here are upcoming sessions, or you can view them on the
...
-  - June 21
-  - July 19
   - August 16
+  - September 20
+  - October 18
 - **AMER (Americas)**: every third Thursday of the month at 9AM PST (GMT -8),
   join
   [here](https://us06web.zoom.us/j/87037874951?pwd=WGo3eUZpeWFZTlhJQXhJeXZhQmwvUT09)
```
Comment (chalin) [inline content/en/community/end-user/discussion-group.md:47]:
```suggestion
- **AMER (Americas)**: every third Thursday of the month at 9AM PST (GMT -8),
  ([Zoom link](https://us06web.zoom.us/j/87037874951?pwd=WGo3eUZpeWFZTlhJQXhJeXZhQmwvUT09))
```

File: content/en/community/end-user/discussion-group.md [hunk 4]
```diff
@@ -32,22 +32,21 @@ Here are upcoming sessions, or you can view them on the
 [OpenTelemetry calendar](https://github.com/open-telemetry/community#calendar):
 
 - **EMEA (Europe, Middle East, and Africa)**: every third Tuesday of the month
-  at 11AM CET (GMT +1), join
-  [here](https://us06web.zoom.us/j/85691064809?pwd=c0VCejh)
-  - June 20
-  - July 18
+  at 11AM GMT, join [here](https://us06web.zoom.us/j/85691064809?pwd=c0VCejh)
```
Comment (chalin) [inline content/en/community/end-user/discussion-group.md:35]:
```suggestion
  at 11AM GMT ([Zoom link](https://us06web.zoom.us/j/85691064809?pwd=c0VCejh))
```

File: content/en/community/end-user/discussion-group.md (hunk 1 above)
Comment (chalin) [reply content/en/community/end-user/discussion-group.md]: Reapplied suggested edit following the fix:format commit

**Review by cartermp — APPROVED** [review]

**Pull request merged**